### DISCUSSION TITLE
Transfer ownership (without credit card)

### DISCUSSION THREAD

**Discussion by jonlucasoffice** [vercel/community#5936] [Account Issue] [answered]
### Question

I would like to transfer ownership of a project to the client. 

When I create a new team, Vercel prevents me from adding the client to the project, and asks for me for my payment info. 

Is there any way to transfer ownership to the client, and have them pay the Pro fee?

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out! I would normally recommend creating a new team with both you and your client, but it sounds like you already tried that and ran into problems. Another option is to have your client to invite you to their team until the project is transferred.

https://vercel.com/docs/projects/overview#transferring-a-project
